### ISSUE TITLE
Evaluate Models

### ISSUE THREAD

**Issue by lsobieski** [Temple-University-CIS-4496/cis-4496-project-hvac#10]
- Utilize a validation set separate from training set during training to prevent overfitting 

 -  Use cross validation depending on model runtime
